### PR TITLE
Api endpoint

### PR DESCRIPTION
**Pull request by seanhamlin**
Add a new endpoint URL to Sumologic, to allow you to use other data centers for Sumologic.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -17,18 +17,11 @@ class Client {
   const QUERY_COMPLETE = 4;
   const QUERY_JOB_CANCELLED = 5;
 
-  private $queryStatusDefinitions = [
```
Comment (fiasco) [inline src/Client.php]: Removing this doesn't seem related?

**Review by fiasco — COMMENTED** [review]
Like the API change, just not sure why the `$queryStatusDefinitions`  was removed. Even if its not used, might be useful to use it at some stage.

File: src/Client.php (hunk 1 above)
Comment (seanhamlin) [reply src/Client.php]: PHPstorm just complained that this var is not being used at all.

Comment (seanhamlin) [reply src/Client.php]: Can remove it from the PR, if you want, or if you have bigger plans for this not-used var.

**Comment by seanhamlin** [comment]
Stripped the code style commits, now it is just the API change.
